e2e_tests/test_standalone/test_study_list.py

```diff
@@ -20,3 +23,46 @@ def test_home(
     title = element.text_content()
     assert title is not None
     assert title == "Optuna Dashboard (Wasm ver.)"
+
+
+def test_load_storage(
+    page: Page,
+    server_url: str,
+) -> None:
+    current_dir = os.getcwd()
+    study_name = "single-objective"
+    url = f"{server_url}"
+
+    def create_storage_file(filename: str):
+        import optuna
+
+        storage = optuna.storages.RDBStorage(f"sqlite:///{filename}")
+        study = optuna.create_study(study_name=study_name, storage=storage)
+
+        def objective(trial: optuna.Trial) -> float:
+            x1 = trial.suggest_float("x1", 0, 10)
+            x2 = trial.suggest_float("x2", 0, 10)
+            return (x1 - 2) ** 2 + (x2 - 5) ** 2
+
+        study.optimize(objective, n_trials=100)
+
+    with tempfile.NamedTemporaryFile() as fp:
+        filename = fp.name
+        path = os.path.join(current_dir, filename)
+        create_storage_file(filename)
+        page.goto(url)
+        with page.expect_file_chooser() as fc_info:
+            page.get_by_role(
+                "button",
+                name="Load an Optuna Storage Drag your SQLite3 file here or click to browse.",
+            ).click()
+        file_chooser = fc_info.value
+        file_chooser.set_files(path)
+
+        page.get_by_role("link", name=study_name).click()
+
+        element = page.query_selector(".MuiTypography-root")
+        assert element is not None
+        title = element.text_content()
+        assert title is not None
+        assert title == "Optuna Dashboard"
```

Comment:
How about checking the number of `<Card>` components that shows the study names?